lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
I think it should always be the case to prevent configurations like this: `multiline-ternary: [2, 'always', 'blah']`. I don't think it's worth adding the fixes for other rules in this PR, but we should probably create a new issue to track this. @btmills Could you correct me if I'm wrong?